dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/OAuth20EndpointServices.java

```diff
@@ -342,7 +342,15 @@ protected void handleEndpointRequest(HttpServletRequest request,
                         "com.ibm.ws.security.oauth20.web.OAuth20EndpointServices", "324", this);
             }
             boolean suppressBasicAuthChallenge = isBrowserWithBasicAuth; // ui must NOT log in using basic auth, so logout function will work.
-            WebUtils.sendErrorJSON(response, e.getHttpStatus(), e.getErrorCode(), e.getErrorDescription(), suppressBasicAuthChallenge);
+            /*
+             * BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_MALFORMED_REQUEST");
+             * Tr.error(tc, errorMsg.getServerErrorMessage());
+             * throw new OidcServerException(errorMsg.getBrowserErrorMessage(), OIDCConstants.ERROR_INVALID_CLIENT_METADATA, HttpServletResponse.SC_BAD_REQUEST);
+             */
+
+            // BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(e.getBrowserServerLogMessage().getTraceComponent(), request.getLocales(), e.getBrowserServerLogMessage().getMessageKey());
+            // e.getBrowserServerLogMessage().setTraceComponent(tc);
```

Comment:
I know it's planned, but did just want to make sure this commented out code gets removed before you deliver.

Reply:
Reminder to get rid of all of this commented out code.